webapp/src/components/expanded_view/call_participant.tsx

```diff
@@ -56,40 +57,36 @@ export default function CallParticipant(props: Props) {
                         alignItems: 'center',
                         bottom: 0,
                         right: 0,
-                        background: 'rgba(50, 50, 50, 1)',
+                        background: props.isMuted ? 'var(--calls-badge-bg)' : '#3DB887',
                         borderRadius: '30px',
                         width: '20px',
                         height: '20px',
                     }}
                 >
                     <MuteIcon
-                        fill={props.isMuted ? '#C4C4C4' : '#3DB887'}
+                        fill='white'
                         style={{width: '14px', height: '14px'}}
-                        stroke={props.isMuted ? '#C4C4C4' : ''}
                     />
                 </div>
                 {props.isHandRaised &&
-                    <>
-                        <div style={styles.reactionBackground}/>
-                        <div style={styles.handRaisedContainer}>
-                            <CompassIcon
-                                icon={'hand-right'}
-                                style={{marginRight: '2px'}}
-                            />
-                        </div>
-                    </>
+                <div style={styles.handRaisedContainer}>
+                    <HandEmoji
+                        style={{
+                            fill: 'rgb(255, 188, 66)',
```

Comment:
Should this be awayIndicator?

Reply:
I've never doubled checked that and assumed it was some custom yellow color but you are right they are the same. Will update, thanks!